UKHomeOffice/sgmr-service
# 1.1.2 SPCR-350 Fix spelling mistake in responsible person form

## Description
The forms label says County but should be Country (the error message if not completed does read Country).

Also includes some minor linting fixes

## To Test
- Sign in
- Create a new voyage plan
- Go to skipper address page
- Country label should now be spelled correctly (was previously county)

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
